h/services/profile_group.py

```diff
@@ -7,7 +7,7 @@ class ProfileGroupService(object):
 
     """A service for retrieving groups associated with users."""
 
-    def __init__(self, session, route_url, open_group_finder):
+    def __init__(self, session, request_authority, route_url, open_group_finder):
```

Comment:
I'm a fan of `request_authority` as a variable name :+1: That's much clearer than just `authority` for example (which I think is probably used for the request authority elsewhere in the code) because which authority is it?